Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -233,8 +233,15 @@ public class PBXProjGenerator {
             .sorted { $0.nameOrPath.localizedStandardCompare($1.nameOrPath) == .orderedAscending }
             .map { $0 }
 
+        let assetTags = Array(Set(project.targets
+            .map { target in
+                target.sources.map { $0.resourceTags }.flatMap { $0 }
+            }.flatMap { $0 }
+        ))
```

Comment:
Let's sort them as well, so it's deterministic. If you add a `sorted()` on the end, then the Array initializer isn't needed either, as sorted returns an array

Reply:
2c81e80